src/Umbraco.Core/PropertyEditors/PropertyValueConverterBase.cs

```diff
@@ -48,7 +50,29 @@ public virtual PropertyCacheLevel GetPropertyCacheLevel(IPublishedPropertyType p
 
     /// <inheritdoc />
     public virtual object? ConvertIntermediateToXPath(IPublishedElement owner, IPublishedPropertyType propertyType, PropertyCacheLevel referenceCacheLevel, object? inter, bool preview)
-        => inter?.ToString() ?? string.Empty;
+    {
+        var d = new XmlDocument();
+        XmlElement e = d.CreateElement("values");
+        d.AppendChild(e);
+
+        if (inter is IEnumerable<string> collection)
+        {
+            foreach (var value in collection)
+            {
+                XmlElement ee = d.CreateElement("value");
+                ee.InnerText = value;
+                e.AppendChild(ee);
+            }
+        }
+        else
+        {
+            XmlElement ee = d.CreateElement("value");
+            ee.InnerText = inter?.ToString() ?? string.Empty;
+            e.AppendChild(ee);
+        }
+
+        return d.CreateNavigator();
```

Comment:
See my comment on https://github.com/umbraco/Umbraco-CMS/issues/6431#issuecomment-535495273 about changing this to `e.CreateNavigator()` to avoid having to write `//*[firstNames/values/value='John']`, but instead a much nicer `//*[firstNames/value='John']`...

@nul800sebastiaan I don't believe this change is already shipped, right? If so, we should apply the above fix and it should at least be in a next minor, since it can change the XPath querying behaviour (it's kind of a functional breaking change, because you could potentially get different results with this fix applied).

Reply:
@ronaldbarendse as far as me and @callumbwhyte could see it, this is not breaking anything, it's actually only enabling something that never worked in the first place. Asking for `ConvertIntermediateToXPath` on a property of type `IEnumerable<string>` would always return `System.String[]`, which is useless. 

I'm afraid we already have the exact same implementation in https://github.com/umbraco/Umbraco-CMS/blob/contrib/src/Umbraco.Core/PropertyEditors/ValueConverters/MultipleTextStringValueConverter.cs#L77 which does `d.CreateNavigator()` so making this one `e.CreateNavigator()` would be inconsistent.

Reply:
But this changes the base implementation, so any PVC that doesn't have an override and would produce a valid value for `ToString()` will now contain a different XPath value/iterator (using the values/value elements)... I'm not sure whether this is actually the case for any of the core PVCs, so might not even be a real issue, but wanted to at least point that out 😉

I'm fine with keeping this PR as-is, especially since we're planning on removing XPath support in a future version anyway: https://github.com/umbraco/Announcements/issues/10 😇